projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/band.model.ts

```diff
@@ -0,0 +1,65 @@
+import { Color, TimeDuration } from '@hypertrace/common';
+import {
+  BOOLEAN_PROPERTY,
+  Model,
+  ModelApi,
+  ModelProperty,
+  NUMBER_PROPERTY,
+  STRING_PROPERTY
+} from '@hypertrace/hyperdash';
+import { ModelInject, MODEL_API } from '@hypertrace/hyperdash-angular';
+import { Observable } from 'rxjs';
+
+@Model({
+  type: 'band'
+})
+export class BandModel<TInterval> {
+  public static readonly BAND_COLOR: string = Color.Gray2;
+  public static readonly BASELINE_COLOR: string = Color.Gray4;
+  public static readonly BASELINE_NAME: string = 'Baseline';
+  public static readonly UPPER_BOUND_NAME: string = 'Upper Bound';
+  public static readonly LOWER_BOUND_NAME: string = 'Lower Bound';
+  public static readonly DEFAULT_OPACITY: number = 0.4;
+
+  @ModelProperty({
+    key: 'band-color',
+    type: STRING_PROPERTY.type
+  })
+  public bandColor: string = BandModel.BAND_COLOR;
+
+  @ModelProperty({
+    key: 'baseline-color',
```

Comment:
So cycling back to my [comment](https://github.com/hypertrace/hypertrace-ui/pull/535#discussion_r565387690) about the expected json, I think the last strange thing is that the band is implicitly providing the baseline. I'm ok if you want to come back to that.

Reply:
Yeah, while I agree that the current way is a little strange I don't really like explicitly calling out the baseline as another series either since its attached to the data call of the band. Maybe we should have just named `bands` something like `baselines` instead.

Let's revisit this one.

Reply:
> another series either since its attached to the data call of the band.

This is the part I was referring to about separating in one of my other comments. They access different data, they should be separate specs (that get merged for perf).

Reply:
I must be missing something here.

The `baseline` and the `lower` and `upper` bounds come from the same single `baselineSeries` call. This is why I say they are associated. But I do still agree from a chart perspective there is more affinity between the baseline and the actual metric since they are charted as `Series`, while the lower and upper bounds are charted as `Band` so agree the json model is confusing in that respect.

Reply:
So imagine a request for two different timeseries with different aggregations of the same metric. Say, p99 and p50 like this example from the service overview:
```typescript
 {
                    type: 'series',
                    name: 'p99',
                    'visualization-type': 'line',
                    color: Color.BlueGray4,
                    data: {
                      type: 'entity-metric-timeseries-data-source',
                      metric: {
                        type: 'metric-timeseries',
                        metric: 'duration',
                        aggregation: MetricAggregationType.P99
                      }
                    }
                  },
                  {
                    type: 'series',
                    name: 'p50',
                    'visualization-type': 'line',
                    color: Color.Blue3,
                    data: {
                      type: 'entity-metric-timeseries-data-source',
                      metric: {
                        type: 'metric-timeseries',
                        metric: 'duration',
                        aggregation: MetricAggregationType.P50
                      }
                    }
                  },
```

These are different series, specified differently, even though they're both asking for data thats contained within the series of a particular entity + metric. Merging them is the responsibility of the request merger (which could use some love). These `value` portion of the baselineSeries and the upper/lower bounds should work the same way in my mind. Completely separate requests as far as the model and specs are concerned.